tests/sample/cgroup_sock_addr2.c

```diff
@@ -41,7 +41,7 @@ update_audit_map_entry(bpf_sock_addr_t* ctx)
 {
     uint64_t key = 0;
     sock_addr_audit_entry_t entry = {0};
-    entry.process_id = bpf_sock_addr_get_current_pid_tgid(ctx);
```

Comment:
nit: should we keep test for the older helper also, as it still has some usage?

Reply:
No, as this function is deprecated.